core/module.rbs

```diff
@@ -1629,5 +1629,7 @@ class Module < Object
   # `attr_reader(name)` but deprecated. Returns an array of defined method names
   # as symbols.
   #
-  def attr: (*Object::name arg0) -> Array[Symbol]
+  def attr: (*id arg0) -> Array[Symbol]
+
+  type id = Symbol | string
```

Comment:
`Object::name`?

Reply:
Yes, It's very similar to `Object::name`.
However, I am questioning the correctness of `Object::name` name.
The expression is ambiguous with `name`.
The context here is `Module`, and `id` is used inside CRuby, as in `rb_check_id`.

Reply:
Should those using `Object::name` switch to `Module::id` instead?